symengine/polys/upolybase.h

```diff
@@ -64,9 +61,15 @@ inline integer_class to_integer_class(const piranha::integer &x)
 }
 #endif
 
-inline integer_class to_integer_class(flint::fmpzxx_srcref i)
+inline integer_class to_integer_class(const fz_t& i)
 {
-    return integer_class(i._data().inner);
+    return integer_class(i.get_fmpz_t());
+}
+
+#else
+inline integer_class to_integer_class(const integer_class &i)
```

Comment:
This need to be outside the `if/endif` directives. Should be available at all times, the initial place looked alright?

Agree with the `inline const integer_class&` -> `inline integer_class` change.